src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -72,15 +120,25 @@ class StacktraceLink extends AsyncComponent<Props, State> {
     return null;
   }
   renderMatchNoUrl() {
-    //TODO: Improve UI
-    return <OpenInContainer columnQuantity={2}>No Match</OpenInContainer>;
+    const {config} = this.match;
+    const {organization} = this.props;
+    const text = this.errorText;
+    const url = `/settings/${organization.slug}/integrations/${config?.provider.key}/${config?.integrationId}/?tab=codeMappings`;
```

Comment:
@MeredithAnya I am wondering if we want to open directly to the code mapping and have the popup open. Might be overkill though. This approach is fine with me just curious what you think.

Reply:
I thought about it, I decided to keep this for now. I think maybe that could be good going forward.